README.md

```diff
@@ -66,6 +67,18 @@ An example of this can be found in the [Decentraland repository](https://github.
     yarn build      # Will drop the results in dist/
     yarn build-ipfs # Will also deploy to IPFS and output an IPFS hash
     ```
+9.  Deploy your subgraph to one of the hosted nodes. The following command builds and deploys the subgraph continuously as you are making changes to it:
```

Comment:
Hosted nodes are not a thing that's generally available. It would be better if this said "Deploy your subgraph to a Graph Node".